examples/todomvc/kusk-backend-api.yaml

```diff
@@ -155,6 +155,7 @@ spec:
               application/json:
                 schema:
                   type: object
+                  additionalProperties: false
```

Comment:
I think we should add the comment to this field explaining what it does to the users.